Fixes for Hilla launch (#1080)
vaadin/docs: articles/application/state-management.asciidoc

> When using Lit that means you should extend the `MobxLitElement` class instead of `LitElement`.

Reviewer: ```suggestion
When using Lit, you need to extend the `MobxLitElement` class instead of `LitElement`.
```
